crates/sui-types/src/messages_consensus.rs

```diff
@@ -221,26 +221,20 @@ impl ConsensusTransactionKind {
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[allow(clippy::large_enum_variant)]
 pub enum VersionedDkgMessage {
-    V0(dkg_v0::Message<bls12381::G2Element, bls12381::G2Element>),
+    V0(), // deprecated
     V1(dkg_v1::Message<bls12381::G2Element, bls12381::G2Element>),
 }
 
 #[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub enum VersionedDkgConfimation {
-    V0(dkg::Confirmation<bls12381::G2Element>),
+pub enum VersionedDkgConfirmation {
+    V0(), // deprecated
```

Comment:
as with the database types, not sure i understand the need to change the old versions